src/apps/vpn/ui/screens/subscriptionNeeded/ViewSubscriptionNeeded.qml

```diff
@@ -0,0 +1,194 @@
+        anchors.left: parent.left
+        anchors.right: parent.right
+
+        spacing: 0
+
+        Image {
+            id: logo
+
+            Layout.topMargin: isMobile ? MZTheme.theme.vSpacing : window.height * 0.2
+            Layout.alignment: Qt.AlignHCenter
+
+            source: "qrc:/ui/resources/logo-connecting.svg"
+            smooth: true
+            antialiasing: true
+        }
+
+        MZHeadline {
+            id: headline
+
+            Layout.fillWidth: true
+            Layout.topMargin: MZTheme.theme.vSpacing
+            Layout.leftMargin: MZTheme.theme.windowMargin * 1.5
+            Layout.rightMargin: MZTheme.theme.windowMargin * 1.5
+
+            text: MZI18n.PurchaseWebTitle
+        }
+
+        MZTextBlock {
+            Layout.fillWidth: true
+            Layout.leftMargin: MZTheme.theme.windowMargin * 1.5
+            Layout.rightMargin: MZTheme.theme.windowMargin * 1.5
+
+            text: MZI18n.PurchaseWebMessage
+                .arg("<b style='color:" + MZTheme.theme.fontColorDark + ";'>"
+                + VPNUser.email + "</b>")
+            font.pixelSize: MZTheme.theme.fontSize
+            horizontalAlignment: Text.AlignHCenter
+            textFormat: Text.RichText
+            wrapMode: Text.WordWrap
+            lineHeight: 22
+
+            Accessible.role: Accessible.StaticText
+            // Prevent html tags from being read by screen readers
+            Accessible.name: text.replace(/<[^>]*>/g, "")
+        }
+
+        Loader {
+            Layout.topMargin: MZTheme.theme.vSpacing
+            Layout.leftMargin: MZTheme.theme.windowMargin
+            Layout.rightMargin: MZTheme.theme.windowMargin
+            Layout.fillWidth: true
+            Layout.maximumWidth: MZTheme.theme.maxHorizontalContentWidth
+            Layout.alignment: Qt.AlignHCenter
+
+            active: vpnFlickable.isMobile
+            sourceComponent: ColumnLayout {
+                spacing: 16
+
+                ButtonGroup {
+                    id: subscriptionOptions
+                }
+
+                Repeater {
+                    id: productList
+                    model: VPNProducts
+                    delegate: MZSubscriptionOption {}
+                }
+            }
+        }
+
+
+        MZButton {
+            id: subscribeNow
+            objectName: "vpnSubscriptionNeededView"
+
+            text: MZI18n.PurchaseSubscribeNow
+
+            Layout.topMargin: MZTheme.theme.vSpacing
+            Layout.leftMargin: MZTheme.theme.windowMargin * 2
+            Layout.rightMargin: MZTheme.theme.windowMargin * 2
+            Layout.fillWidth: true
+
+            onClicked: isMobile ? VPNPurchase.subscribe(subscriptionOptions.checkedButton.productId) : VPNPurchase.subscribe("web")
+        }
+
+
+        Loader {
+            Layout.fillWidth: true
+            Layout.topMargin: MZTheme.theme.vSpacingSmall
+            Layout.alignment: Qt.AlignHCenter
+
+            active: vpnFlickable.isMobile
```

Comment:
@peterbenvenuto the mocks show the ToS and Privacy Policy links as mobile only, but they are currently on desktop as well. Should they remain visible on desktop?

Reply:
Spoke with @peterbenvenuto - ToS and Privacy Policy links should be present on all platforms. Mocks have been update and `isMobile` requirement has been lifted